### PR TITLE
fix-three-types

### PR DESCRIPTION
**Pull request by alexx855**
https://github.com/makimenko/angular-template-for-threejs/issues/367

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Click below to see the demo in a web browser:<br>
 ## Usage in your Angular project
 
 1. Set up an Angular project and install dependencies:  
-   `npm i three three.meshline atft --save`
+   `npm i three @types/three three.meshline atft dagre @types/dagre  --save`
```
Comment (makimenko) [inline README.md:48]: mesline removed

**Review by makimenko — COMMENTED** [review]
- There are some conflicts in `package-lock.json` and `workstation-actor.component.ts`, please pull latest version
- Build failing, it seems that types in `src/app/...` should be fixed (TS2322: Type 'string' is not assignable to type 'number')

**Comment by alexx855** [comment]
@makimenko you have any idea why the storybook error?

`SyntaxError: Unexpected token } in JSON at position 4070`

File: projects/atft/src/lib/actor/data-center/index.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export * from './atft-data-center-actor.module';
+// export * from './atft-data-center-actor.module'; Duplicated export, in public_api.ts
```
Comment (makimenko) [inline projects/atft/src/lib/actor/data-center/index.ts:1]: Better to export here, but remove from public.ts

**Review by makimenko — COMMENTED** [review]
please remove tableAndChar.dae from assets (not used, but failing on new build)

File: angular.json [hunk 3]
```diff
@@ -131,7 +135,7 @@
           "builder": "@angular-devkit/build-angular:ng-packagr",
           "options": {
             "tsConfig": "projects/atft/tsconfig.lib.json",
-            "project": "projects/atft/ng-package.json"
+            "project": "projects/atft/ng-package.json",
```
Comment (makimenko) [inline angular.json:138]: trailing comma causing storybook failure (found comment in stackoverflow)

Comment (alexx855) [reply angular.json:138]: thanks, will try to catch this in the IDE before fix it

**Comment by makimenko** [comment]
@alexx855 Thanks for contribution!

**Pull request merged**